app.css

```diff
@@ -3,3 +3,24 @@
 @import url("/auth/auth.css");
 @import url("/product/product.css");
 @import url("/wishlist/wishlist.css");
+@import url("/cart/cart.css");
+
+/* Scrollbar */
+::-webkit-scrollbar {
+  width: 6px;
+  height: 6px;
+}
+
+::-webkit-scrollbar-track {
+  background: transparent;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: var(--color-tertiary-dark);
+  border-radius: 100px;
+  transition: background-color 0.3s;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background-color: var(--color-grey-dark-2);
```

Comment:
try to give generic name to color varibales. The purpose of color variables is that variable remains same only color can be changed in future. If you name the variable based on color name, it will create problem.

Reply:
I have used, these aren't generic they are for example say border's , box-shadows, outlines, etc. For other purpose I have used names like `primary, success, error`, etc